src/ui/atoms/button/button.scss

```diff
@@ -40,14 +36,42 @@ $background-colors: (
 
     &.size-small {
       padding: 4px 10px;
+
+      &.icon {
+        padding: 7px;
```

Comment:
There is some inconsistencies between small/medium and large paddings, I will have uniformed paddings more consistently: 8px for small, 12px for medium and 16px for large (keep a 4px step)